apache/couchdb-fauxton
# Update config addon to use redux

## Overview

Update config addon to use redux

## Testing recommendations

- Go to Configuration page
- Click 'Add Option' button, fill in the values and press 'Create'
  - The list is updated and the new option is displayed
- Click on the new option to edit its value. Change the value and press the green button to save.
  - Option value is updated
- Click the Trash button and confirm the deletion
  - List is refreshed and option no longer appears

## Checklist
